configs/platforms/osx-10.10-x86_64.rb

```diff
@@ -3,8 +3,9 @@
   plat.servicedir '/Library/LaunchDaemons'
   plat.codename "yosemite"
 
-  plat.provision_with 'softwareupdate -i "Command Line Tools (OS X 10.10) for Xcode-7.2"'
-  plat.provision_with 'mkdir /usr/local; cd /usr/local; git clone https://github.com/Homebrew/homebrew.git .; git checkout ab475 -- Library/Formula/boost.rb; /usr/local/bin/brew install pkgconfig'
-  plat.install_build_dependencies_with "PATH=$PATH:/usr/local/bin brew install "
+  plat.provision_with 'mkdir /usr/local; curl http://pl-build-tools.delivery.puppetlabs.net/osx/homebrew.tar.gz | tar -x --strip 1 -C /usr/local -f -; /usr/local/bin/brew tap puppetlabs/brew-build-tools http://pl-build-tools.delivery.puppetlabs.net/osx/brew-build-tools.git --full; /usr/local/bin/brew tap-pin puppetlabs/brew-build-tools'
```

Comment:
yeah -- i'll do that.  i thought i had split the line after homebrew gets extracted, but apparently i stopped at the thought.